### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by Juliocho990**
### Summary
- Nav bar was created
- Nav bar was located fixed.

### Screenshot
nav-bar
nav-bar

### Trello card
https://trello.com/c/zdi9q0uZ/31-navbar

### CODE REVIEW

File: list.html [hunk 1]
```diff
@@ -2,69 +2,75 @@
...
 </head>
-<body>
-  <div class="container list">
-    <div class="content">
-      <div class="list__card">
-        <div class="list__cover">
-          <img class="list__cover__image" src="assets/book-cover.png" alt="Wolox book cover">
+<body class="body">
+  <nav class="nav-bar">
+    <a class="logo" href="#home"></a>
```
Comment (eledwinn) [inline list.html]: Page logo is not decorative element. Please add it as <image />

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: list.html (hunk 1 above)
Comment (Juliocho990) [reply list.html]: Ok

File: scss/nav-bar.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  background-color: $white;
+  border-top: 4px solid $deep-sky-blue;
+  box-shadow: 0 4px 10px 0 rgba($black, 0.5);
+  display: flex;
+  height: 70px;
+  justify-content: space-around;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
```
Comment (CrisVega08) [inline scss/nav-bar.scss]:
```suggestion
  z-index: 1;
```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/nav-bar.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/nav-bar.scss]: ok

**Review by eledwinn — APPROVED** [review]

**Pull request merged**